.idea/codeStyles/Project.xml

```diff
@@ -1,336 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
-  <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
-    <option name="IGNORE_SAME_INDENTS_FOR_LANGUAGES" value="true" />
-    <option name="OTHER_INDENT_OPTIONS">
-      <value>
-        <option name="INDENT_SIZE" value="2" />
-        <option name="CONTINUATION_INDENT_SIZE" value="4" />
-        <option name="TAB_SIZE" value="2" />
```

Comment:
Why are we removing this file here?

Reply:
its auto generate file , it just commit by mistake.

Reply:
@MohitMaliFtechiz Please revert this chage

Reply:
@gouri-panda please check now